readme.md

```diff
@@ -1,32 +1,86 @@
 # Microsoft Azure IoT SDKs for Java
 This repository contains the following:
-* **Azure IoT Hub Device SDK**: to connect client devices to Azure IoT Hub (supports Java 7+ and Android API 17+)
-* **Azure IoT Hub Service SDK**: enables developing back-end applications for Azure IoT (supports Java 7+)
+* **Azure IoT Hub device SDK for Java**: to connect client devices to Azure IoT Hub (supports Java 7+ and Android API 17+)
+* **Azure IoT Hub service SDK for Java**: enables developing back-end applications for Azure IoT (supports Java 7+)
+
+The API reference documentation for the device SDK is [here][java-api-reference-device].
+
+The API reference documentation for the service SDK is [here][java-api-reference-service].
 
 To find SDKs in other languages for Azure IoT, please refer o the [azure-iot-sdks][azure-iot-sdks] repository
 
```

Comment:
While you are at it please correct typo - refer to

Reply:
Corrected